includes/snmptrap/authenticationFailure.inc.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+log_event('SNMP Trap: Authentication Failure: ' . $device['sysName'], $device, 'auth', 3, $device['hostname']);
```

Comment:
Logging by device sysName is specific to your env. You either need to log using hostname or run it through `format_hostname()` to get the type the user wants.

Reply:
Yeah. Done!